packages/website/docs/createQuerySuggestionsPlugin.md

```diff
@@ -2,9 +2,35 @@
 id: createQuerySuggestionsPlugin
 ---
 
+The Query Suggestions plugin adds [Algolia Query Suggestions](https://www.algolia.com/doc/guides/building-search-ui/ui-and-ux-patterns/query-suggestions/js/) to your autocomplete.
```

Comment:
Are these changes supposed to be here?

Reply:
Ugh, no, I started from the wrong branch. I just rebased.